compiler-rt/test/asan/TestCases/strdup_oob_test.cpp

```diff
@@ -23,7 +23,7 @@ int main(int argc, char **argv) {
   // CHECK: AddressSanitizer: heap-buffer-overflow
   // CHECK: #0 {{.*}}main {{.*}}strdup_oob_test.cpp:[[@LINE-2]]
   // CHECK-LABEL: allocated by thread T{{.*}} here:
-  // CHECK: #{{[01]}} {{.*}}strdup
+  // CHECK: #{{[0-9]}}+ {{.*}}strdup
```

Comment:
Should the `+` be inside the double braces since I assume you are trying to use it as a regex and not explicitly match it? I believe this is causing a test failure on a bot because of this misplacement of the `+` on https://lab.llvm.org/buildbot/#/builders/77/builds/38204

Reply:
Yeah I had the same change locally but forgot to update the commit ...
Thanks